backend/api/views/annotation_campaign.py

```diff
@@ -159,6 +169,18 @@ def report(self, request, pk=None):
         )
 
         for result in results:
+
+            if campaign.confidence_indicator_set is not None:
+                lvl_max = max(
```

Comment:
I'd prefer `max([for indicator.level for indicator in campaign.confidence_indicator_set.confidence_indicators.all()])` but actually this might be better to impliment as a ConfidenceIndicatorSet model property method so you can just do `campaign.confidence_indicator_set.max_level` here :wink:

Reply:
```python
# What I proposed
max([i.level for i in self.confidence_indicators.all()])
# What you used
max(self.confidence_indicators.all(), key=lambda x: x.level).level
```
Actually I had an extra for in my example that broke it, is that what stopped you from using it ? List comprehension are considered more pythonic and it's a bit shorter than your method, any reason you don't want to use it ?